Sources/TuistAnalytics/Models/CommandEvent.swift

```diff
@@ -7,6 +7,7 @@ public struct CommandEvent: Codable, Equatable, AsyncQueueEvent {
     public let name: String
     public let subcommand: String?
     public let params: [String: String]
+    public let commandArguments: String
```

Comment:
There's some overhead in saving arrays to a database and it's easy to make it back to an array. While I agree, I think it's simpler to keep it as-is.

Reply:
Yeah but that's a backend problem, so I would rather make the mapping to string on the backend before saving to the DB, rather than in the client, WDTY? 🚀

Reply:
yeah, sure, I can update it 🙂

Reply:
Implemented here: https://github.com/tuist/tuist/pull/4383/commits/583017de6ab2110196a402a547ad88d3fa9c6379